pkg/daemon/update.go

```diff
@@ -50,6 +50,39 @@ const (
 	realtimeKernelType = "realtime"
 )
 
+// HostInfo contains information of an OSTree based system
+type HostInfo struct {
+	Deployments []Deployment `json:"deployments"`
+}
+
+// Deployment contains information about a particular OSTree deployment
+type Deployment struct {
+	RequestedLocalPkgs []string `json:"requested-local-packages"`
+}
+
+func installedRTKernelRpmsOnHost() ([]string, error) {
+	var out []byte
+	var err error
+	var rtKernelRpms = []string{}
+	if out, err = exec.Command("rpm-ostree", "status", "--json").Output(); err != nil {
+		wrappedErr := fmt.Errorf("Failed to execute rpm-ostre status --json %v", err)
```

Comment:
You can directly return the fmt.Errorf below without the extra wrapperErr variable

Reply:
Right, fixed it.